### ISSUE TITLE
[Question] Demo website

### ISSUE THREAD

**Issue by Gasjki** [J0hnHawk/FS17_WebStats#71]
Hi. Where can I find a demo of this script ? Thank you!

**Comment by J0hnHawk** [comment]
Hi.
Ich uploaded the script at http://j0hnhawk.bplaced.net/

**Comment by sixxkilur** [comment]
Nice Work; and thank you for the demo site.
![image](https://user-images.githubusercontent.com/4954874/38235760-d869fff6-36e7-11e8-8f69-e04c82d473a4.png)

**Comment by J0hnHawk** [comment]
Thank you. 
Mh, it seems I forgot some translations in the english language file ;-)

**Comment by sixxkilur** [comment]
I am unsure if it is or not the screenshot I provided was from your url with the site set to English; it looks to me that it could be the names used on your save game map for those locations. I have not played FS17 long enough to know much about the other maps yet I have only been tinkering on Goldcrest Valley. I went ahead and took the liberty of looking at your script at the translation files for each language and the search string  that shows up on your site RAIFFEISEN does not exist in any of the global.lng files. My site on the same category only reports this.
![image](https://user-images.githubusercontent.com/4954874/38294590-f34720aa-37b0-11e8-8177-adb5073e1fa2.png)




I am currently working on dark style for Giant's FS17 webStatsSDK not completed as of yet but this is what it looks like at the moment.
![image](https://user-images.githubusercontent.com/4954874/38293117-ec3c38ae-37ab-11e8-91c1-87b77dffdf68.png)

I had it matching your FS17_WebStats style(Farming Simulator2017) but a power outage caused the alterations to be lost. I will repeat color scheme when I am able to get around to it.
